src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -524,15 +524,15 @@ export default {
 			}
 
 			&:last-child {
-				margin-right: var(--default-clickable-area);
+				margin-right: $message-status-icon-width;
 			}
 		}
 	}
 }
 
 .message-status {
-	width: var(--default-clickable-area);
-	height: 24px;
+	width: $message-status-icon-width;
+	height: var(--default-line-height);
 	display: flex;
 	justify-content: center;
```

Comment:
For @ShGKme comment:
```suggestion
	justify-content: flex-end;
```
B | A
-- | --
![image](https://github.com/user-attachments/assets/4e206fe6-771f-49da-8eef-aa4e08ac7af6) | ![image](https://github.com/user-attachments/assets/7fd2a391-1632-4bf9-833d-683d694dd142)